pom.xml

```diff
@@ -50,7 +50,7 @@
     <log4j2.version>2.17.1</log4j2.version>
     <hcommon-hive-metastore.version>1.2.3</hcommon-hive-metastore.version>
     <jackson.version>2.9.8</jackson.version>
-    <jackson.databind.version>2.9.10.8</jackson.databind.version>
+    <jackson.databind.version>2.12.7.1</jackson.databind.version>
```

Comment:
I wouldn't upgrade this without upgrading `jackson.version` above to something similar. So I'd suggest closing this PR and deciding if you want to do a separate PR to upgrade both jackson libs together.